content/guides/hanami/v2.2/command-line/generate.md

```diff
@@ -65,7 +65,7 @@ $ bundle exec hanami generate migration --help
 
 ### hanami generate operation
 
-Generates an [operation](//guide/operations):
+Generates an [operation](//guide/overview):
```

Comment:
This seems like a change that we do want to keep? `//guide/operations` feels like the right destination for this link.